client/src/App.tsx

```diff
@@ -0,0 +1,92 @@
+//import { useState } from 'react'
+//import reactLogo from './assets/react.svg'
+//import viteLogo from '/vite.svg'
+import './App.css'
+import { Event } from "./components/Event/Event.tsx";
+import { WrapperDiv } from "./components/WrapperDiv/WrapperDiv.tsx";
+
+function App() {
+    const birthdayDates = [
```

Comment:
Tohle je treba explicitne otypovat,napr.
```
const birthdayDates: DateRecord[]
```

jinak ti u pouziti na radcich 84 u `<Event` krici typescript, ze ti nesedi typy. Protoze on komponenta ceka, ze `answer` ma jen 3 mozne hodnoty. Ale protoze jsi tady neuvedl typ, TypeScript chape answer jako `string` - tudiz bez omezeni hodnot. A to ted haze chybu :)